core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java

```diff
@@ -11,7 +11,11 @@
 
 package alluxio.security.authorization;
 
-import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
```

Comment:
/usr/src/alluxio/core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java:16: error: Extra newline

Reply:
Thanks, it succeeded this time.